glib/src/gstring.rs

```diff
@@ -1076,6 +1088,18 @@ impl crate::value::ToValue for Vec<GString> {
     }
 }
 
+impl From<Vec<GString>> for Value {
+    fn from(mut v: Vec<GString>) -> Self {
+        unsafe {
+            let mut value = Value::for_value_type::<Vec<GString>>();
+            let ptr: *mut *mut c_char = ToGlibContainerFromSlice::to_glib_full_from_slice(&v);
+            v.set_len(0);
```

Comment:
Why? In the line above you copy the whole strings anyway. Maybe you wanted `to_glib_container_from_slice()` above to steal the strings?

Reply:
No, we need to call `to_glib_full` on each of the strings to force them to a C string

Reply:
Actually no, it should be `into_glib_ptr`, I guess those `GString` implementations should be calling that because they can avoid the copy

Reply:
`into_glib_ptr()` on `Vec<GString>` would steal the strings themselves but store the array in newly allocated memory? Makes sense, yes. That's what I meant above and what you would effectively get by calling `to_glib_container_from_slice()` and truncating the `Vec` without dropping the values

Reply:
Looks like this is done now